### PR TITLE
feat[805]: allow longer text in story description

### PR DESCRIPTION
**Pull request by sveneh**
https://trello.com/c/2f9QAkpj

### CODE REVIEW

File: src/build.js [hunk 1]
```diff
@@ -401,32 +397,44 @@ class Build
     /**
      * ## getDescription
      *
-     * gets the first text description in the content and returns that
+     * gets description from the content and returns that. Either returns full text or only from
+     * the first block
      *
-     * @param {Array} arr array filled w/ content
+     * @param {Array} storyTextArr array filled w/ content
```
Comment (janwilamowski) [inline src/build.js]: I'd rather use the more descriptive `String[]` as type description. There are other options as well: http://usejsdoc.org/tags-type.html

File: src/build.js [hunk 2]
```diff
@@ -401,32 +397,44 @@ class Build
...
+     * @param {Boolean} useFullText when false, use only the first text content block
      * @param {Number} i recursive index
      *
-     * @return {Mixed} text content or false _String or Boolean_
+     * @return {String} text content. Might be empty if no text.
      */
-    getDescription( arr, i = 0 )
+    getDescription( storyTextArr, useFullText = true, i = 0 )
```
Comment (janwilamowski) [inline src/build.js]: don't include the type in the variable name. Instead, call it textSections or something the like

File: src/build.js [hunk 3]
```diff
@@ -232,26 +231,23 @@ class Build
...
+        const paragraph       = create( 'div', classes.PARAGRAPH );
+        description           = this.getDescription( JSON.parse( description ) );
 
-        if ( description )
-        {
-            paragraph.innerHTML = description;
-            paragraph.innerHTML = paragraph.textContent;
-        }
+        paragraph.innerHTML = description;
+        paragraph.innerHTML = paragraph.textContent; // What??
```
Comment (janwilamowski) [inline src/build.js]: this doesn't look right

Comment (sveneh) [inline src/build.js]: exactly, but I didn't know what it was used for either ...

File: src/build.js (hunk 1 above)
Comment (sveneh) [inline src/build.js]: but it's not a string, it's an array of objects (yes, I thought it would be a string, too ...)

File: src/build.js (hunk 2 above)
Comment (sveneh) [inline src/build.js]: can do

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Styla Widget  2.1.11
+Styla Widget  2.1.12
```
Comment (eliasliedholm) [inline README.md]: I normally only do the version bump once the branch has been merged on stage, and stage is ready to be merged on master. Stage is currently already at 2.1.12.

**Pull request merged**